homeassistant/components/ambiclimate/climate.py

```diff
@@ -241,4 +180,12 @@ async def async_update(self):
         if token_info:
             await self._store.async_save(token_info)
 
-        self._data = await self._heater.update_device()
+        data = await self._heater.update_device()
+        self._attr_target_temperature = data.get("target_temperature")
+        self._attr_current_temperature = data.get("temperature")
+        self._attr_current_humidity = data.get("humidity")
+        self._attr_min_temp = self._heater.get_min_temp()
+        self._attr_max_temp = self._heater.get_max_temp()
```

Comment:
Hope I am not annoying, the min & max are set only at start and should be at init only, not here

Reply:
No, min/ max might change depending on the modus (heat/auto/...)

Reply:
Since the implementation only supports heating mode, it is probably ok to do move it to init

Reply:
Figured that so I moved it.

Reply:
> Since the implementation only supports heating mode, it is probably ok to do move it to init
I verified they are not changed before the comment by looking at the external library, they are only updated when `update_device_info` is called which is called only on init:
https://github.com/home-assistant/core/blob/5ca634609c9cfead04756a291f1826466ecf352d/homeassistant/components/ambiclimate/climate.py#L88
https://github.com/Danielhiversen/pyAmbiclimate/blob/c2606b9951bd24965b2921f7c15efdd9909d3ed5/ambiclimate/__init__.py#L266
https://github.com/Danielhiversen/pyAmbiclimate/blob/c2606b9951bd24965b2921f7c15efdd9909d3ed5/ambiclimate/__init__.py#L248